code/datums/martial/sleeping_carp.dm

```diff
@@ -177,9 +175,11 @@
 	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
 	block_chance = 50
+	force_unwielded = 10
+	force_wielded = 24
 
 /obj/item/twohanded/bostaff/update_icon()
-	icon_state = "bostaff[wielded]"
+	icon_state = "bostaff[is_wielded()]"
```

Comment:
uh. I have no idea if this is a okay thing or not. If it works its fine by me.

Reply:
I'm in the process of using signals for all of it on another branch. It does work though, just not preferred.